index.bs

```diff
@@ -285,6 +287,18 @@ Try to construct a table with the columns of the properties of <var>tabularData<
 
 Perform Logger("warn", <var>data</var>).
 
+<h4 id="dir" oldids="dir-data,dom-console-dir" method for="console">dir(<var>data</var>)</h4>
```

Comment:
I'm assuming it is literally a list of old-ids that other specs may refer to, so we tell bikeshed "Hey this thing may also go by these names too {name1, name2, ...}"?